src/components/Ping.vue

```diff
@@ -37,7 +37,19 @@ export default {
   },
   methods: {
     async pingServer () {
-      const response = await fetch(`http://localhost:3000/ping?ping=${this.ping}`)
+      function createBaseUrl () {
+        const locationToMatchRegex = new RegExp('^http://localhost:[0-9]*')
```

Comment:
You should also include : into optional characters: (:[0-9]+)?

Reply:
Done.